nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-api/src/main/java/org/apache/nifi/audit/ProcessGroupAuditor.java

```diff
@@ -371,6 +419,30 @@ private void saveUpdateProcessGroupAction(final String groupId, final Operation
         saveAction(action, logger);
     }
 
+    private void saveUpdateConnectableActions(final List<Connectable> connectables, final Operation operation, final Component component) {
+        final List<Action> actions = new ArrayList<>();
+        for (final Connectable connectable : connectables) {
+             final Action action = generateUpdateConnectableAction(connectable, operation, component);
+             actions.add(action);
+        }
```

Comment:
This can be simplified to
```suggestion
       final List<Action> actions = connectables.stream()
       .map(connectable -> generateUpdateConnectableAction(connectable, operation, component))
.toList();
```